docs/tutorial/part-three/index.md

```diff
@@ -189,7 +189,7 @@ export default ({ children }) => (
 )
 ```
 
-3. Import this new layout component into your `src/pages/index.js` page component:
+3. हा नवीन लेआउट कॉम्पोनन्ट आपल्या `src/pages/index.js` पेज कॉम्पोनन्ट आयात करा:
```

Comment:
Use import,  don't translate

Reply:
Done. Please check.
Thanks for your review!